resources/capabilitystatement-bulk-data.json

```diff
@@ -0,0 +1,117 @@
+{
+  "resourceType": "CapabilityStatement",
+  "id": "bulk-data",
+  "url": "http://hl7.org/fhir/us/bulkdata/CapabilityStatement/bulk-data",
+  "version": "1.0.0",
+  "name": "BulkDataIGCapabilityStatement",
+  "title": " FHIR Bulk Data Export (Flat FHIR) Implementation Guide CapabilityStatement",
+  "status": "active",
+  "experimental": false,
+  "date": "2019-06-04T00:00:00+00:00",
+  "publisher": "SMART Health IT",
+  "contact": [
+    {
+      "name": "Ricky Sahu",
+      "telecom": [
+        {
+          "system": "email",
+          "value": "ricky@1up.health"
+        }
+      ]
+    },
+    {
+      "name": "Dan Gottlieb",
+      "telecom": [
+        {
+          "system": "email",
+          "value": "daniel.gottlieb@childrens.harvard.edu"
+        }
+      ]
+    },
+    {
+      "name": "Josh Mandel",
+      "telecom": [
+        {
+          "system": "email",
+          "value": "joshua.mandel@childrens.harvard.edu"
+        }
+      ]
+    },
+    {
+      "name": "Vlad Ignatov",
+      "telecom": [
+        {
+          "system": "email",
+          "value": "Vladimir.Ignatov@childrens.harvard.edu"
+        }
+      ]
+    }
+  ],
+  "description": "This Section describes the expected capabilities of a FHIR Bulk Data Server actor which is responsible for providing responses to the queries submitted by the FHIR Bulk Data Requestors. The complete list of FHIR profiles, RESTful operations, and search parameters supported by FHIR Bulk Data Servers are defined. Systems implementing this capability statement should meet the requirements set by the [Bulk Data Implementation Guide](https://fhir.org/ig/HL7/bulk-data/). FHIR Bulk Data Clients have the option of choosing from this list to access necessary data based on their local use cases and other contextual requirements.",
+  "jurisdiction": [
+    {
+      "coding": [
+        {
+          "system": "http://unstats.un.org/unsd/methods/m49/m49.htm",
+          "code": "001",
+          "display": "World"
+        }
+      ]
+    }
+  ],
+  "kind": "requirements",
+  "fhirVersion": "4.0.0",
+  "format": [
+    "json"
+  ],
+  "patchFormat": [
+    "application/json-patch+json"
+  ],
+  "implementationGuide": [
+    "http://hl7.org/fhir/us/core/ImplementationGuide/hl7.fhir.bulk.data"
```

Comment:
I'm still expecting this to look like `http://hl7.org/fhir/bulk-data/ImplementationGuide/bulk-data`

Reply:
updated here https://github.com/HL7/bulk-data/pull/8/files#diff-7ca1fcb8dc53e609e860ea10f6c5e443